pulp-in-one-container.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Pulp in One Container
+sidebar: home_sidebar
+permalink: /pulp-in-one-container/
+toc: false
+---
+
+Installing Pulp 3 and getting all the services running can be challenging. To reduce the complexity of getting started with Pulp, the Pulp team created a single container image that has all necessary services to run Pulp 3.
+
+The image is available under the `pulp` namespace on [Dockerhub](https://hub.docker.com/repository/docker/pulp/pulp-fedora31/). This image includes the Ansible, Container, File, Maven, and RPM plugins. A new version is published every time there is a plugin update available. You can update your environment to the latest version of the container using `docker pull`.
+
+The following commands can be used to start Pulp 3.4.1 with `podman 1.8.0`, which is available on Fedora 31:
```

Comment:
You should also mention that it works with docker client also. The commands are the same except 'podman' is replaced with 'docker'.